digital_land/load.py

```diff
@@ -32,23 +32,32 @@ def resource_hash_from(path):
 
 
 class DictReaderInjectResource(csv.DictReader):
-    def __init__(self, resource, *args, **kwargs):
+    def __init__(self, resource, include_line_num, *args, **kwargs):
         self.resource = resource
+        self.include_line_num = include_line_num
+        if include_line_num:
+            self.line_num = 1
```

Comment:
Does this need updating after the convo on Slack ?

Reply:
no this starts counting from 1 (ignoring header)